### ISSUE TITLE
Height of nav bar in S screens is too tall

### ISSUE THREAD

**Issue by yaili** [canonical-websites/www.ubuntu.com#1695]
Can it match live? Reducing the font size in L screens might help

**Comment by barrymcgee** [comment]
@yaili Is the Vanilla navigation pattern also too tall?

**Comment by yaili** [comment]
No, the Vanilla pattern looks right. I see it as 46px tall when the viewport is 414px wide, whereas ubuntu.com is showing as 54px tall.

**Comment by pmahnke** [comment]
@yaili is this still an issue?

**Comment by yaili** [comment]
@pmahnke Yes, but I don't know where best to report it:
- ubuntu.com changes heights between viewports
- vanillaframework.io keeps height between viewports
- neither of these two sites is following the [Vanilla spec](https://github.com/ubuntudesign/vanilla-design/blob/master/Navigation/navigation.md)
- I don't know where the issue is, because it might be that one or both are not using the latest Vanilla, but it the problem is that Vanilla itself isn't following the spec this issue should be moved to vanilla-framework

@kwm14 for reference ^

**Comment by deadlight** [comment]
Whoever picks this up will need to get some up-to-date design input on the latest version of the site.

**Comment by deadlight** [comment]
@kwm14 I think that you're the best person to clarify the current state of this issue and the next-steps.
